tests/Integration/Behaviour/Features/Scenario/Discount/BO/add_cart_level_discount.feature

```diff
@@ -99,7 +78,7 @@ Feature: Add discount
       | valid_to          | 2019-12-01 00:00:00 |
       | code              | PROMO_CART_2019_4   |
       | reduction_percent | -10.0               |
-    Then I should get error that discount field reduction_percent is invalid
+    Then I should get error that discount field reduction_amount is invalid
```

Comment:
🤔 something doesn't seem logical here You provide a reduction **percent** but it's the reduction **amount** that is invalid..